tests/python/gaia-ui-tests/gaiatest/tests/graphics/orientation_zoom_base.py

```diff
@@ -67,34 +71,14 @@ def orientation_zoom_check(self):
         self.take_screenshot()
 
         # go back and forth with flicking then exit gallery app
-        self.scroll(image._current_frame_locator, 'right', 150)
-        self.take_screenshot()
-        self.scroll(image._current_frame_locator, 'left', 150)
+        GaiaImageCompareTestCase.scroll(self.marionette, 'right',
+                                        150, locator=image._current_frame_locator)
+
         self.take_screenshot()
-        self.apps.kill(gallery.app)
-        time.sleep(2)
+        GaiaImageCompareTestCase.scroll(self.marionette, 'left',
+                                        150, locator=image._current_frame_locator)
         self.take_screenshot()
 
-        # Launch browser.  Go to Mozilla FirefoxOS site
-        # Scroll up/down, change orientation, scroll up/down
-        # commented out due to bug 1127324
-
-        # search = Search(self.marionette)
-        # search.launch()
-        # browser = search.go_to_url('http://mozilla.org/firefoxos')
-        # browser.wait_for_page_to_load()
-        # browser.switch_to_content()
-        #
-        # self.take_screenshot()
-        # self.marionette.switch_to_frame()
-        # self.scroll(browser._browser_frame_locator, 'up', 400)
-        # self.take_screenshot()
-        # self.change_orientation('landscape-primary')
-        # self.scroll(browser._browser_frame_locator, 'down', 300)
-        # self.take_screenshot()
-        # self.scroll(browser._browser_frame_locator, 'down', 300)
-        # self.take_screenshot()
-
```

Comment:
Initially it was commented out for a while, because of the scroll action not working in browser.  Then I added the scroll feature on test_browser_navigation.py, but I didn't remove this line because it was already commented out.

Interestingly, the scroll action on browser is broken again - I think there is a bug for it already, but if not I should raise one